playbooks/openstack/advanced-configuration.md

```diff
@@ -327,17 +327,34 @@ The `openshift_openstack_required_packages` variable also provides a list of the
 prerequisite packages to be installed before to deploy an OpenShift cluster.
 Those are ignored though, if the `manage_packages: False`.
 
-## Multi-master configuration
```

Comment:
Doesn't the multi-master configuration case require anything to be noted in the docs any more?

Reply:
I don't think so. Do you have anything concrete in mind?

Reply:
I meant, anything *specific to the multi-master*. IIUC, The cloud-native LBaaS has nothing to multi-master, right?

Reply:
> I don't think so. Do you have anything concrete in mind?

Expanded in the comments below, wrt the default solution for openshift-ansible. Seems like not supported from now on, right?